packages/terra-i18n/src/translations/de.js

```diff
@@ -0,0 +1,31 @@
+import { addLocaleData } from 'react-intl';
+import localeData from 'react-intl/locale-data/de';
+
+addLocaleData(localeData);
+
+const messages = {
+  'Terra.ajax.error': 'Inhalt konnte nicht geladen werden.',
```

Comment:
Do we want to include any translations at this time? Are any components using these translations?